### ISSUE TITLE
sed: -e expression #1, char 50: unterminated `s' command

### ISSUE THREAD

**Issue by flyemsafe** [Qubinode/qubinode-installer#32]
NETMASK_PREFIX listed twice

**Comment by flyemsafe** [comment]
This issue still exist in the current developer branch.

It appears when `qubinode-installer -m dns` is executed.

```
sed: -e expression #1, char 48: unterminated `s' command
Configuring DNS server
```

**Comment by flyemsafe** [comment]
fixed 5b3614896f5d7a8bf01811c4e517716ffbd7b3de
